src/admin-views/admin-welcome-message.php

```diff
@@ -39,6 +39,8 @@ class="tribe-events-admin-title__logo"
 		</ul>
 	</div>
 
+	<?php tribe( \TEC\Common\Admin\Conditional_Content\Black_Friday::class )->render_narrow_banner_html(); ?>
```

Comment:
Can we simplify the FQN?

```suggestion
	<?php tribe( Black_Friday::class )->render_narrow_banner_html(); ?>
```